### PR TITLE
D8CORE-2531: Fixing the header for the people node

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Making sure there is a Contact header for all the nodes. If there is no content then the header is gone.

# Review By (Date)
- 9/28

# Criticality
- High.
- All the sites using people

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Update configs
3. Verify there is a Contact header on a person page. It will have the telephone icon to the left side. It is not related to the telephone content. Try adding and removing content. If you remove all content from the right column, the header should disappear.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [x] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- All products using the Person node

# Associated Issues and/or People
- D8CORE-2531
- https://github.com/SU-SWS/stanford_profile/pull/302

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: config/install/core.entity_view_display.node.stanford_person.default.yml [hunk 1]
```diff
@@ -766,6 +766,3 @@ content:
 hidden:
   layout_builder__layout: true
   su_person_research_interests: true
-  su_person_academic_appt: true
-  su_person_admin_appts: true
-  su_person_scholarly_interests: true
```
Comment (sherakama) [inline config/install/core.entity_view_display.node.stanford_person.default.yml:771]: These should probably stay.

File: templates/sections/section-person-body-two-column.html.twig [hunk 2]
```diff
@@ -11,6 +11,9 @@
 
   {% if content.right|render %}
     <div {{ region_attributes.right.addClass('right-region', right_class) }}>
+      <div class="node-stanford-person-su-person-contact">
+        <h2 class="block__title">Contact</h2>
+      </div>
```
Comment (sherakama) [inline templates/sections/section-person-body-two-column.html.twig:16]: Do we want to make this conditional on anything being in the `content.right` container or is always displaying ok?

Comment (jenbreese) [reply templates/sections/section-person-body-two-column.html.twig:16]: We do have the `{% if content.right|render %}` and if there is no other content, the header doesn't appear. If there is any content, it should appear. It worked ok when I tested it locally. Can you think of a reason/case where it wouldn't?

File: config/install/core.entity_view_display.node.stanford_person.default.yml (hunk 1 above)
Comment (jenbreese) [reply config/install/core.entity_view_display.node.stanford_person.default.yml:771]: Ok. What is the best way to get them back? Do I add the lines manually back?

File: templates/sections/section-person-body-two-column.html.twig (hunk 2 above)
Comment (sherakama) [reply templates/sections/section-person-body-two-column.html.twig:16]: Good enough for me.

File: config/install/core.entity_view_display.node.stanford_person.default.yml (hunk 1 above)
Comment (sherakama) [reply config/install/core.entity_view_display.node.stanford_person.default.yml:771]: That would be the easy way. Sure.

Comment (jenbreese) [reply config/install/core.entity_view_display.node.stanford_person.default.yml]: 👍 Ok. I did it that way

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**